tests/integration/components/blog-post-test.js

```diff
@@ -0,0 +1,27 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+
+moduleForComponent('blog-post', 'Integration | Component | blog post', {
+  integration: true
+});
+
+test('it renders', function(assert) {
+  assert.expect(2);
+
+  // Set any properties with this.set('myProperty', 'value');
+  // Handle any actions with this.on('myAction', function(val) { ... });
+
+  this.render(hbs`{{blog-post}}`);
+
+  assert.equal(this.$().text(), '');
+
+  // Template block usage:
+  this.render(hbs`
+    {{#blog-post}}
+      template block text
```

Comment:
Expected ')' and instead saw 'template'.<br>Missing semicolon.<br>Expected an assignment or function call and instead saw an expression.